Writers/SqlWriter.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.IO;
+using System.Text.RegularExpressions;
+using EntityManager.CodeGenerators;
+using EntityManager.ViewModels;
+
+namespace EntityManager.Writers
+{
+    public class SqlWriter : IWriter
+    {
+        public string SqlFilePath { get; private set; }
+
+        public SqlWriter(string sqlFilePath)
+        {
+            this.SqlFilePath = sqlFilePath;
+        }
+
+        public void Write(ShellViewModel model)
+        {
+            string fileContents;
+            using (StreamReader reader = new StreamReader(SqlFilePath))
+            {
+                fileContents = reader.ReadToEnd();
+                reader.Close();
+            }
```

Comment:
if you use File.WriteAllText at the and, is it not better to use File.ReadAllText in here?

Reply:
Caused errors of file being busy when at WriteAllText line